app/views/dashboard/_sidebar.html.erb

```diff
@@ -1,5 +1,57 @@
-<br />
+  <br />
+ 	<div class="panel panel-default">
+	  <div class="panel-heading">
+	    <h1 class="panel-title">Ask a question</h1>
+	  </div>
+	  <div class="panel-body">
+	    <p>The easiest way to start with Public Lab community is to ask a question.<br />See questions and ask your own here:</p>
```

Comment:
I think this could be two separate `<p>`s instead of using a `br` -- do you think you could make that small change? Thanks!!!